api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -209,10 +209,12 @@ type FelixConfigurationSpec struct {
 
 	// VXLANEnabled overrides whether Felix should create the VXLAN tunnel device for VXLAN networking. Optional as Felix determines this based on the existing IP pools. [Default: nil (unset)]
 	VXLANEnabled *bool `json:"vxlanEnabled,omitempty" confignamev1:"VXLANEnabled"`
-	// VXLANMTU is the MTU to set on the tunnel device. See Configuring MTU [Default: 1440]
-	VXLANMTU  *int `json:"vxlanMTU,omitempty"`
-	VXLANPort *int `json:"vxlanPort,omitempty"`
-	VXLANVNI  *int `json:"vxlanVNI,omitempty"`
+	// VXLANMTU is the MTU to set on the IPv4 VXLAN tunnel device. See Configuring MTU [Default: 1440]
+	VXLANMTU *int `json:"vxlanMTU,omitempty"`
+	// VXLANV6MTU is the MTU to set on the IPv6 VXLAN tunnel device. See Configuring MTU [Default: 1420]
+	VXLANV6MTU *int `json:"vxlanV6MTU,omitempty"`
```

Comment:
I think this should be `VXLANMTUV6` - like this field: https://github.com/projectcalico/calico/blob/6dc63ea2c84e9ac4e858bc27e5e8efb3f5f2cfc6/api/pkg/apis/projectcalico/v3/felixconfig.go#L310

Reply:
Will do! Thanks!